verification/.devcontainer/devcontainer.json

```diff
@@ -12,13 +12,12 @@
             "extensions": [
                 "uavcan.dsdl",
                 "wholroyd.jinja",
-                "ban.spellright",
-                "ms-python.python",
-                "ms-vscode.cpptools",
                 "ms-vscode.cpptools-extension-pack",
+                "streetsidesoftware.code-spell-checker",
                 "xaver.clang-format",
-                "hbenl.vscode-test-explorer",
-                "matepek.vscode-catch2-test-adapter"
+                "vadimcn.vscode-lldb",
+                "matepek.vscode-catch2-test-adapter",
```

Comment:
What do we need catch2 support here for?

Reply:
It's just a bad id. This support googletest:

```
Name: C++ TestMate
Id: matepek.vscode-catch2-test-adapter
Description: Run GoogleTest, Catch2 and DOCtest tests from VSCode
Version: 4.4.2
Publisher: Mate Pek
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=matepek.vscode-catch2-test-adapter
```